### PR TITLE
Orthanc dicom server

### PR DESCRIPTION
**Pull request by CGDogan**
I'm not touching keycloak and pathdb, because they seem to be related to permission handling and the dicom server needs a completely different auth setup.

### CODE REVIEW

File: caMicroscope.yml [hunk 1]
```diff
@@ -58,3 +58,23 @@ services:
...
+    restart: unless-stopped
+    stdin_open: true
+    tty: true
+    ports:
+      - "8042:8042"
+      - "11112:11112"
+    volumes:
+      - ./jwt_keys/:/root/keys/
+      - ./images/:/images/
+      - ./OrthancConfiguration.json:/root/src/Configuration.json
```
Comment (birm) [inline caMicroscope.yml:76]: I think you need to check in OrthancConfiguration.json

**Comment by CGDogan** [comment]
The default DICOM configuration is to allow everything. Should I mention it in the README?

**Comment by CGDogan** [comment]
Sorry @birm actually I remembered that to test Orthanc, you would need to change slideloader and caracal to point to my pull requests https://github.com/camicroscope/SlideLoader/pull/84 https://github.com/camicroscope/Caracal/pull/163  `https://github.com/cgdogan/caracal.git#fs_update` `https://github.com/cgdogan/slideloader.git#dicom`

**Review by birm — APPROVED** [review]
end to end test looks good. It'll be nice when we can collapse the file information to avoid having such a long table when using dicoms.

**Pull request merged**